src/locales/messages.pot

```diff
@@ -2933,6 +2942,9 @@ msgstr ""
 msgid "WAGMI!"
 msgstr ""
 
+msgid "Wallet Address"
```

Comment:
**suggestion**: maybe we can use `Wallet address` instead, to relieve burden on translators (unless there's a reason to capitalize Address)

Reply:
Where even is "Wallet address"? A search int he code doesn't find it